CefSharp/Handler/IRenderProcessMessageHandler.cs

```diff
@@ -14,7 +14,7 @@ public interface IRenderProcessMessageHandler
         /// OnContextCreated is called in the Render process immediately after a CefV8Context is created.
         /// An IPC message is immediately sent to notify the context has been created
         /// (should be safe to execute javascript). If the page has no javascript then no V8Context will be created
-        /// and as a result this method will not be called. Currently only called for the Main frame <see cref="IFrame.IsMain"/>
```

Comment:
Will to mention this will be called for every `v8context` and to check `IFrame.IsMain` to determine if the it's the main context.